back/db/seeds/citizenlab.rb

```diff
@@ -554,7 +547,9 @@ def create_for_current_locales
 open_idea_project.set_default_topics!
 
 # Create settings for Home Page.
-HomePage.create!
+HomePage.create!({
+  header_bg: Rails.root.join('spec/fixtures/header.jpg').open
```

Comment:
Seeds the OS version with our default banner image, as before we implemented the new `HomePage` model.